dandiapi/urls.py

```diff
@@ -80,6 +81,9 @@ def to_url(self, value):
     ),
     path('api/users/me/', users_me_view),
     path('api/users/search/', users_search_view),
+    # TODO remove these endpoints after migration
+    path('api/scripts/map_ids/', map_ids_view, name='map-ids'),
+    re_path(r'^api/scripts/map_ids/(?P<dandiset_id>[0-9]{6})/$', map_id_view, name='map-id'),
     path('accounts/', include('allauth.urls')),
```

Comment:
This might make it a little more clear:

```suggestion
    path('api/users/search/', users_search_view),
    
    # TODO remove these endpoints after migration
    path('api/scripts/map_ids/', map_ids_view, name='map-ids'),
    re_path(r'^api/scripts/map_ids/(?P<dandiset_id>[0-9]{6})/$', map_id_view, name='map-id'),
    
    path('accounts/', include('allauth.urls')),
```

Reply:
This violates linting, sadly